packages/twenty-server/src/modules/workflow/workflow-executor/workspace-services/workflow-executor.workspace-service.ts

```diff
@@ -75,13 +102,11 @@ export class WorkflowExecutorWorkspaceService {
         currentStepIndex,
         steps,
         payload,
+        output,
```

Comment:
I think it would be interesting to also store the output of the failed attempts. See my comment where I suggest adding two more fields to WorkflowExecutorOutput.